atd-vze/src/queries/tempRecords.js

```diff
@@ -3,14 +3,54 @@ import { gql } from "apollo-boost";
 export const GET_TEMP_RECORDS = gql`
   query getTempRecords {
     crashes(
-      where: { is_temp_record: { _eq: true } }
-      order_by: { record_locator: desc }
+      where: {
+        is_temp_record: { _eq: true }
+        _and: { is_deleted: { _eq: false } }
+      }
     ) {
+      id
       record_locator
       case_id
       crash_timestamp
       updated_by
       updated_at
+      units {
+        id
+      }
+    }
+  }
+`;
+
+export const SOFT_DELETE_TEMP_UNITS_CRASH = gql`
```

Comment:
i could break these up into two separate operations if thats less confusing, but i figure the less code the better?